data/applications.yml

```diff
@@ -148,6 +148,7 @@
 - github_repo_name: asset-manager
   type: APIs
   team: "#2ndline"
+  puppet_name: asset-manager
```

Comment:
This would break the link to the puppet config. I think the correct solution would be to use the `app_name` in the deploy dashboard URL:

https://github.com/alphagov/govuk-developer-docs/blob/master/app/app_docs.rb#L82-L84

Looks like that matches a lot better to the list used here:

https://github.com/alphagov/govuk-puppet/blob/ed07581c5054c8ef9183e4101571f6cd2fe5f60e/hieradata_aws/common.yaml#L873-L985

Reply:
Aha. I did wonder whether adding the puppet_name might have unintended side effects. I'll take another look at this.